Density/app/src/main/java/org/cornelldti/density/density/facilitydetail/FacilityInfoPage.kt

```diff
@@ -276,9 +271,12 @@ class FacilityInfoPage : BaseActivity() {
                     opHoursTimestamps = hoursTimeStampsList
                     setOpenOrClosedOnToolbar()
                 },
-                facilityHoursStringsOnResponse = {
+
+                facilityHoursStringsOnResponse = { opHoursStringsList ->
+                    opHoursStrings = opHoursStringsList
                     fetchMenuJSON(FluxUtil.getCurrentDate(), facilityClass!!.id)
-                })
+                }
+        )
```

Comment:
this doesn't work - we only call `fetchOperatingHours` once, so it won't change when you change the days

Reply:
I'm pretty sure we've always wanted the top menu to remain the same tho, right? Like that should always show today's open/closed data

Reply:
ohh yes good catch